index.html

```diff
@@ -1,3 +1,40 @@
-﻿
+<!doctype html>
+<html lang="en">
+<head>
+  <meta name="description" content="Demo project">
+  <title>The Best Chocolate Chip Cookies</title>
+</head>
+<body>
+  <h1>The Best Chocolate Chip Cookies</h1>
+    <section>
+      Recipe by Grandma
+      Prep Time: 45 Min
```

Comment:
Notice that although your text is on two lines in the html file, they show up on 1 line in the browser: https://ramnah.github.io/2019-spring-01h-recipe-page/

To separate the lines you could wrap each line in a `p` tag.

Sorry if this is a spoiler, but you can also add the image with `<img src='img/cookies.jpg' />`